module/batch-action/src/Domain/Command/ProcessBatchActionResourceCommand.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Domain\Command;
+
+use Ergonode\BatchAction\Domain\Entity\BatchActionId;
+use Ergonode\SharedKernel\Domain\AggregateId;
+use Ergonode\EventSourcing\Infrastructure\DomainCommandInterface;
+
+class ProcessBatchActionResourceCommand implements DomainCommandInterface
```

Comment:
Shouldn't it be ProcessBatchActionEntryCommand? Entry combines **resource identifier** and **action** which should be performed on it.